src/NHibernate/AdoNet/IParameterAdjuster.cs

```diff
@@ -0,0 +1,11 @@
+using System.Data.Common;
+using NHibernate.SqlTypes;
+using NHibernate.Type;
+
+namespace NHibernate.AdoNet
+{
+	internal interface IParameterAdjuster
```

Comment:
Why is it `internal`? An external driver implementor may need it.

Reply:
It was a prototype, so I kept it from "the public". I wouldn't mind making it public.